core/src/preferences/PreferencesBackend.cpp

```diff
@@ -196,8 +211,63 @@ namespace ledger {
             _cipher = Option<AESCipher>::NONE;
         }
 
+        void PreferencesBackend::resetEncryption(
+            const std::shared_ptr<api::RandomNumberGenerator>& rng,
+            const std::string& newPassword,
```

Comment:
I thought you changed the order, no ? or just fixed your tests ? 
Because it feel more natural to start with old password then new password ! IMHO

Reply:
I just fixed the tests, yep. I’ll switch the order!

![](https://vignette.wikia.nocookie.net/fr.starwars/images/5/52/Base_enneig%C3%A9_Premier_Ordre.jpg/revision/latest?cb=20150417125210)